src/moments.jl

```diff
@@ -363,3 +363,40 @@ end
 
 kurtosis(v::RealArray) = kurtosis(v, mean(v))
 kurtosis(v::RealArray, wv::AbstractWeights) = kurtosis(v, wv, mean(v, wv))
+
+"""
+    cumulant(v, k, [wv::AbstractWeights], m=mean(v))
+
+Return the `k`th order cumulant of a real-valued array `v`, optionally
+specifying a weighting vector `wv` and a pre-computed mean `m`. If `k` is a `UnitRange`, then return all the cumulants of orders in this range as a vector.
```

Comment:
AFAICT any range of integers works, no reason to require a `UnitRange`.
```suggestion
specifying a weighting vector `wv` and a pre-computed mean `m`.

If `k` is a range of `Integer`s, then return all the cumulants of orders in this range as a vector.
```